client/src/components/InterpreterSignUpForm.js

```diff
@@ -158,7 +192,18 @@ class InterpreterSignUpForm extends Component {
                     required
                   />
                 </Form.Field>
-
+                {this.state.hasErrors ? (
+                  <div>
+                    <Message negative>
```

Comment:
According to the ticket, the message should be displayed between the title and the form, not after the form.